application/mongo_client.py

```diff
@@ -0,0 +1,51 @@
+from bson.objectid import ObjectId
+from datetime import datetime
+import os
+import pymongo
+
+from application.item import Item
+
+class MongoClient:
+
+    def __init__(self):
+
+        self.MONGO_USERNAME = os.environ.get('MONGO_USERNAME')
+        self.MONGO_PASSWORD = os.environ.get('MONGO_PASSWORD')
+        self.MONGO_HOST = os.environ.get('MONGO_HOST')
+        self.MONGO_TODO_APP_DATABASE = os.environ.get('MONGO_TODO_APP_DATABASE')
+        self.client = pymongo.MongoClient(f"mongodb+srv://{self.MONGO_USERNAME}:{self.MONGO_PASSWORD}@{self.MONGO_HOST}/?retryWrites=true&w=majority")
+        self.db = self.client[self.MONGO_TODO_APP_DATABASE]
+
+        self.refresh_lists()
+        self.refresh_items()
+
+
+    def refresh_lists(self):
```

Comment:
I think I've said elsewhere, but this refresh pattern is a bit of an anti-pattern because you could be serving users outdated content unless the developer remembers to refresh at the appropriate times. If instead we changed items to a property that fetches them fresh every time we'd remove that risk. Obviously there might be occasions where fetching items fresh isn't always best, but I think here the cost isn't particularly high